compiler/bindings/python/CMakeLists.txt

```diff
@@ -129,6 +129,7 @@ declare_mlir_python_sources(IREECompilerAPIPythonTools
     tools/import_onnx/__main__.py
     tools/ir_tool/__main__.py
     tools/scripts/ireec/__main__.py
```

Comment:
Unrelated to this PR, but we had a TODO somewhere to remove this in favor of just `iree-compile`.

* https://github.com/iree-org/iree/blob/main/compiler/bindings/python/iree/compiler/tools/scripts/ireec/__main__.py
* https://github.com/iree-org/iree/blob/d7e8bcca82678ac63fd79d6d848d556c2196e006/compiler/setup.py#L456-L459

Looks like we should rename the script to `iree_compile/__main__.py`? Then we can drop the alias.

Reply:
yeah, almost removed that but didn't want to double duty in one PR.